datumaro/components/annotation.py

```diff
@@ -557,6 +557,21 @@ def get_bbox(self) -> Tuple[float, float, float, float]:
         y1 = max(ys)
         return [x0, y0, x1 - x0, y1 - y0]
 
+    def get_points(self) -> Optional[List[Tuple[float, float]]]:
+        """
+        Return points as a list of tuples, e.g. [(x0, y0), (x1, y1), ...]
+        """
+        points = self.points
+        if not points:
+            return None
+
+        assert len(points) % 2 == 0, "points should have (2 x points) number of float values."
```

Comment:
To my best knowledge, the difference between 'assert' and 'exception' is that 'assert' will not raise an exception when \_\_debug\_\_ is false.
Thus, I guess that raising an 'exception' may be more suitable in this case.

Reply:
Since `Annotations.points` is normally created by Datumaro's internal logic (not from user inputs), we can generally expect that `len(points) % 2 == 0` almost always. Therefore, I decided to use `assert` keyword (as you mentioned, we use `assert` keyword commonly in this situation).